### DISCUSSION TITLE
Commercial License Documentation

### DISCUSSION THREAD

**Discussion by ambirdev** [emgucv/emgucv#829] [General]
We recently purchased the commercial license and are trying to integrate it into our existing projects.   We are trying to use the documentation here:

https://www.emgu.com/wiki/index.php/Download_And_Installation#Commercial_Release

And the file structure in the delivered files don't seem to match up with the documentation.   Can anyone clarify if we are misinterpreting the documentation or is the documentation stale with the 4.6 release and need to be updated?

**Comment by emgucv** [comment]
It will be the easiest to use the nuget pakcage from the commercial repository. 

If you are adding ".dll" files to your project, and finding structure mismatches, can you point out which files are mismatched?

**Reply by ambirdev** [reply]
Which nuget packages are the commercial Windows CUDA and non-CUDA packages?

**Comment by emgucv** [comment]
"Emgu.CV.runtime.windows.cuda.dldt" is the commercial cuda nuget package.
"Emgu.CV.runtime.windows.dldt" is the commercial none-cuda nuget package

**Reply by ambirdev** [reply]
If you search on those packages in the Visual Studio Nuget Package Manager you get back an empty search result.  And nuget online returns an empty as well:

https://www.nuget.org/packages?q=Emgu.CV.runtime.windows.dldt

**Reply by emgucv** [reply]
The nuget packages are not available from nuget.org, it is hosted from our commercial nuget repo. If you follow the steps here:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Commercial_Release

You will find the instruction for adding the commercial nuget repository into visual studio.

If you need more help setting that up, please send an email to support@emgu.com and we will be able to help from there.
